components/match2/commons/match_summary_base.lua

```diff
@@ -575,6 +579,56 @@ function MatchSummary.addVodsToFooter(match, footer)
 	return footer
 end
 
+---Creates a match footer with vods if vods are set
+---@param match table
+---@return string
+function MatchSummary.createSubstitutesComment(match)
+	local comment = {}
+	Array.forEach(match.opponents, function(opponent)
+		local substitutions = (opponent.extradata or {}).substitutions
+		if Logic.isEmpty(substitutions) then
+			return
+		end
+
+		Array.forEach(substitutions, function(substitution)
+			if Logic.isEmpty(substitution.substitute) then
+				return
+			end
+
+			local subString = {}
+			table.insert(subString, string.format('%s stands in',
+				tostring(PlayerDisplay.InlinePlayer{player = substitution.substitute})
+			))
+
+			if Logic.isNotEmpty(substitution.player) then
+				table.insert(subString, string.format('for %s',
+					tostring(PlayerDisplay.InlinePlayer{player = substitution.player})
+				))
+			end
+
+			if opponent.type == Opponent.team then
+				local team = require('Module:Team').queryRaw(opponent.template)
+				if team then
+					table.insert(subString, string.format('on <b>%s</b>', Page.makeInternalLink(team.shortname, team.page)))
+				end
+			end
+
+			if Table.isNotEmpty(substitution.games) then
+				local gamesNoun = 'map' .. (#substitution.games > 1 and 's' or '')
+				table.insert(subString, string.format('on %s %s', gamesNoun, mw.text.listToText(substitution.games)))
+			end
+
+			if String.isNotEmpty(substitution.reason) then
+				table.insert(subString, string.format('due to %s', substitution.reason))
+			end
+
+			table.insert(comment, table.concat(subString, ' ') .. '.')
+		end)
+	end)
+
+	return table.concat(comment, tostring(Break():create()))
```

Comment:
```suggestion
	if Logic.isEmpty(comment) then return end

	return table.concat(comment, tostring(Break():create()))
```

return nil if no comment is to be set instead of an empty one